### DISCUSSION TITLE
ToS & Privacy Policy on self hosted server

### DISCUSSION THREAD

**Discussion by yucore-22** [dani-garcia/vaultwarden#1772] [General]
Hi,

We are testing Vaultwarden for a couple of weeks now as a candidate for a centralized passwortmanager solution in our company.
Everything works fine so far during the Testphase.

But one of my collogues noticed the Terms of Services and the Privacy Policy you must agree on sign up for an Account.
These links are redirecting to the Bitwarden website and it's Terms of Service and Privacy regulations.

- How these regulations have an effect, if first we selfhost the server and second the project has no association to the original Bitwarden Software and the Company Bitwarden Inc. / 8bit Solutions LLC, as mentioned in the repo description?
- Are these only remaining's of  the original Bitwarden frontend and will be removed in future?

As my Company has very strict privacy policies and we need to know this, before we can use this in production.

Thanks for any input on this.

**Comment by jjlin** [comment]
These agreements are not supposed to be shown for self-hosted installations. They will be removed via dani-garcia/bw_web_builds#40 at some point. If you need it sooner, you can patch and build the web vault yourself.
